frontend/src/concepts/pipelines/elyra/utils.ts

```diff
@@ -105,3 +118,56 @@ export const generateElyraServiceAccountRoleBinding = (
     },
   ],
 });
+
+export const createElyraServiceAccountRoleBinding = async (
+  notebook: NotebookKind,
+): Promise<RoleBindingKind | void> => {
+  const notebookName = notebook.metadata.name;
+  const namespace = notebook.metadata.namespace;
+  const notebookUid = notebook.metadata.uid;
+
+  // Check if rolebinding is already exists for backward compatibility
+  const roleBinding = await getRoleBinding(
+    namespace,
+    getElyraServiceAccountRoleBindingName(notebookName),
+  ).catch((e) => {
+    // 404 is not an error
+    if (e.statusObject?.code !== 404) {
+      // eslint-disable-next-line no-console
+      console.error(
+        `Could not get rolebinding to service account for notebook, ${notebookName}; Reason ${e.message}`,
+      );
+    }
+    return undefined;
+  });
+
+  if (notebookUid) {
+    if (roleBinding) {
+      const ownerReferences = roleBinding.metadata.ownerReferences || [];
+      if (!ownerReferences.find((ownerReference) => ownerReference.uid === notebookUid)) {
+        ownerReferences.push(getElyraRoleBindingOwnerRef(notebookName, notebookUid));
+      }
+      return patchRoleBindingOwnerRef(
+        roleBinding.metadata.name,
+        roleBinding.metadata.namespace,
+        ownerReferences,
+      ).catch((e) => {
+        // This is not ideal, but it shouldn't impact the starting of the notebook. Let us log it, and mute the error
+        // eslint-disable-next-line no-console
+        console.error(
+          `Could not patch rolebinding to service account for notebook, ${notebookName}; Reason ${e.message}`,
```

Comment:
should this throw an error here? otherwise we are hiding a possible issue

Reply:
I think this is the current implementation as it's said in the comment, it's not ideal but shouldn't impact the starting of the notebook so we just log and mute the error.